code/modules/admin/verbs/pray.dm

```diff
@@ -33,11 +33,17 @@
 		prayer_type = "CULTIST PRAYER"
 		deity = "Nar'Sie"
 	else if(isliving(usr))
-		var/mob/living/L = usr
-		if(HAS_TRAIT(L, TRAIT_SPIRITUAL))
+		var/mob/living/prayer = usr
+		if(HAS_TRAIT(prayer, TRAIT_SPIRITUAL))
 			cross.icon_state = "holylight"
 			font_color = "blue"
 			prayer_type = "SPIRITUAL PRAYER"
+	else if(isliving(usr))
```

Comment:
Unreachable code

Reply:
yep, thank you

Reply:
Oddly, this worked in testing. I'll still fix the code.